Dockerfile

```diff
@@ -20,18 +20,18 @@ ENV PIP_NO_CACHE_DIR=yes \
     PYTHONUNBUFFERED=1 \
     PYTHONDONTWRITEBYTECODE=1
 
-# Create a non-root user and set permissions
-RUN useradd --create-home appuser
-WORKDIR /home/appuser
-RUN chown appuser:appuser /home/appuser
-USER appuser
-
-# Copy the requirements.txt file and install the requirements
-COPY --chown=appuser:appuser requirements.txt .
+# Install the required python packages globally
+ENV PATH="$PATH:/root/.local/bin"
+COPY requirements.txt .
 RUN sed -i '/Items below this point will not be included in the Docker Image/,$d' requirements.txt && \
-	pip install --no-cache-dir --user -r requirements.txt
+	pip install --no-cache-dir -r requirements.txt
+
+# Create a non-root user
+RUN useradd -u 1000 --create-home appuser
```

Comment:
there was a user who this was causing folder permissions issues for, so we reverted it back to simply using root.